scripts/test.sh

```diff
@@ -0,0 +1,4 @@
+GREN_GITHUB_TOKEN="$(vault read -field=value secret/mobile/release_token)"
```

Comment:
This file needs to be renamed, otherwise it will create confusion with actual app testing.

The use of this is to test the configuration of tools used to produce a new release.
I suppose that `release-dry-run.sh` is a more appropriate name.
It will also make sense to add other release configuration tests here then.